.github/actions/javascript/reviewerChecklist/index.js

```diff
@@ -42,10 +44,17 @@ function getNumberOfItemsFromReviewerChecklist() {
  */
 function checkIssueForCompletedChecklist(numberOfChecklistItems) {
     GitHubUtils.getAllReviewComments(issue)
-        .then(reviewComments => combinedComments.push(...reviewComments))
+        .then((reviewComments) => {
+            console.log('Pulled all review comments, now adding them to the list...');
```

Comment:
Let's add the number of comments found to the log:
```suggestion
            console.log(`Pulled ${reviewComments.length} review comments, now adding them to the list...`);
```

Same goes for the normal comments below.